core/test/matrix/coo.cpp

```diff
@@ -152,6 +152,26 @@ TYPED_TEST(Coo, CanBeCreatedFromExistingData)
 }
 
 
+TYPED_TEST(Coo, CanBeCreatedFromExistingConstData)
+{
+    using value_type = typename TestFixture::value_type;
+    using index_type = typename TestFixture::index_type;
+    value_type values[] = {1.0, 2.0, 3.0, 4.0};
+    index_type col_idxs[] = {0, 1, 1, 0};
+    index_type row_idxs[] = {0, 0, 1, 2};
+
+    auto mtx = gko::matrix::Coo<value_type, index_type>::create_const(
```

Comment:
It could potentially be handled, but I would prefer to be more explicit, since `create` is a variadic template, which makes for unpleasant error messages if something doesn't fit

Reply:
I think the overload here would make it easier to switch between const and non const arrays without having to "re-write" the code. So, maybe have both ?

Reply:
Good point, I need to think this through a bit more.

Reply:
I think the problem would be the return type. The current `create_const` method returns a `uniqe_ptr<const Coo>`, which the normal `create` doesn't. I don't think it would be possible to get the normal `create` to return both types. So, I'm fine with being a bit more explicit.